provider/cmd/pulumi-resource-aws/main.go

```diff
@@ -19,16 +19,27 @@ package main
 import (
 	"context"
 	_ "embed"
+	"os"
 
 	aws "github.com/pulumi/pulumi-aws/provider/v6"
 	pf "github.com/pulumi/pulumi-terraform-bridge/pf/tfbridge"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/cmdutil"
 )
 
 //go:embed schema-embed.json
 var pulumiSchema []byte
 
+//go:embed schema-light-embed.json
+var pulumiSchemaLight []byte
+
 func main() {
 	ctx := context.Background()
 	info := aws.Provider()
-	pf.MainWithMuxer(ctx, "aws", *info, pulumiSchema)
+
+	s := pulumiSchema
+	if cmdutil.IsTruthy(os.Getenv("PULUMI_AWS_LIGHT_SCHEMA")) {
```

Comment:
Probably, "Light" isn't the optimal name for external. "Correct"? :) "Clean"? Or just "Experimental"?

Reply:
Oof, this one is interesting. Naming is hard. I can be convinced but none of these strike me as quite right.

Correct - makes me think "wait, so the prod schema is incorrect??"

Clean - "wait, so the prod schema is dirty??"

Experimental - "what is the experiment exactly?"

"Minimal" perhaps as it excludes information pertinent to legacy Node-only overlays. More suggestions welcome.